english/dragons/fire-everywhere.md

```diff
@@ -0,0 +1,41 @@
+Three massive dragons appear.
+You know you have seen them somewhere. You try to recall where..
+Where did I see these dragons ?
+Where... Where....Where
+
+The mushroom walls have a smirk on their face..
+They know the answer and they also know that you don't have much time to think...
+
+The walls are getting closer...
+
+Suddenly you remember... These are the [dragons](http://gameofthrones.wikia.com/wiki/Dragons) of the Khaleesi from the Game Of Thrones
```

Comment:
This line is longer than 120 characters. Could you break it up into multiple lines? That way it will be easier to read the raw files or view diffs on the command line.  No need to add blank lines in between, just press enter somewhere in the middle of the line.